### DISCUSSION TITLE
How to open new non-empty tab as first child

### DISCUSSION THREAD

**Discussion by Kuba314** [piroor/treestyletab#3314] [Q&A] [answered]
Hi!

After middle-clicking a link or right-clicking and selecting "Open Link in New Tab", I'd like the new tab to be opened as a first child tab.

Opening a new empty tab (ctrl+T) already does this for me. I'd like non-empty tabs to behave the same way. Sadly, I can't seem to find the right checkbox for this.

Current beaviour:
```
Parent:
  \- old child
  \- new child
```
What I'd like to see:
```
Parent:
  \- new child
  \- old child
```

**Comment by irvinm** [comment, marked as answer]
Under "Tree Behavior":

![image](https://user-images.githubusercontent.com/979729/228041677-8f757212-8e06-425d-87d2-52557b4d991e.png)

Other:

![image](https://user-images.githubusercontent.com/979729/228042250-9ca0f812-d951-4139-9f37-ed8385f38947.png)

I already have this behavior by setting the default position "next to the current tab" by setting the Firefox about:config parm:

![image](https://user-images.githubusercontent.com/979729/228042512-71db3769-11b9-4514-9d72-4a20e633cde7.png)

**Reply by Kuba314** [reply]
Thank you so much! It was the first option you mentioned. I couldn't find it before. It works now!
